run.sh

```diff
@@ -25,153 +28,37 @@ if [ ${#ES_NODE_SIGNER_PRIVATE_KEY} -ne 64 ]; then
   exit 1
 fi
 
-# ZK prover mode, 1: one proof per sample, 2: one proof for multiple samples.
-zkp_mode=
-i=1
-while [ $i -le $# ]; do
-    if [ "${!i}" = "--miner.zk-prover-mode" ]; then
-        j=$((i+1))
-        zkp_mode="${!j}"
-        break
-    else
-        if echo "${!i}" | grep -qE -- "--miner\.zk-prover-mode=([0-9]+)"; then
-            zkp_mode=$(echo "${!i}" | sed -E 's/.*=([0-9]+)/\1/')
-            break
-        fi
-    fi
-    i=$((i+1))
-done
-
-if [ -n "$zkp_mode" ] && [ "$zkp_mode" != 1 ] && [ "$zkp_mode" != 2 ]; then
-  echo "Error: zk prover mode can only be 1 or 2."
-  exit 1  
-fi 
+executable="./build/bin/es-node"
+echo "========== build info =================="
+$executable --version
+echo "========================================"
 
-if [ -n "$zkp_mode" ]; then
-  echo "The zk prover mode has been overridden to $zkp_mode"
-fi 
+data_dir="./es-data"
+storage_file_0="$data_dir/shard-0.dat"
 
-# download zkey if not yet
-zkey_name="blob_poseidon2.zkey"
-zkey_size=560301223
-zkey_url="https://es-node-zkey.s3.us-west-1.amazonaws.com/blob_poseidon2_testnet1.zkey"
-if [ "$zkp_mode" = 1 ]; then
-  zkey_name="blob_poseidon.zkey"
-  zkey_size=280151245
-  zkey_url="https://drive.usercontent.google.com/download?id=1ZLfhYeCXMnbk6wUiBADRAn1mZ8MI_zg-&export=download&confirm=t&uuid=16ddcd58-2498-4d65-8931-934df3d0065c"
-fi
-zkey_file="./build/bin/snark_lib/$zkey_name"
-if [ ! -e  ${zkey_file} ] || [ $(wc -c <  ${zkey_file}) -ne ${zkey_size} ]; then
-  echo "Start downloading ${zkey_file}..." 
-  curl $zkey_url -o ${zkey_file}
-  if [ ! -e  ${zkey_file} ]; then
-    echo "Error: The zkey file was not downloaded. Please try again."
-    exit 1
-  fi
-  if [ $(wc -c <  ${zkey_file}) -ne ${zkey_size} ]; then
-    echo "Error: The zkey file was not downloaded correctly. You can check the file content for more information."
+if [ ! -e $storage_file_0 ]; then
```

Comment:
Do we also want to check the zkey file here?

Reply:
Will leave both checks to the node, e.g.,

```
 CRIT [08-08|10:32:24.330] Init ZK prover failed                    error="zkey does not exist" dir=/Users/dl/code/es-node/build/bin/snark_lib/zkey/blob_poseidon2.zkey
```